suite-native/module-device-settings/src/components/DevicePinActionButton.tsx

```diff
@@ -102,12 +107,14 @@ export const DevicePinActionButton = ({
         });
 
         const { param, successMessageKey, canceledMessageKey } = actionConfigMap[type];
+
         const result = await TrezorConnect.changePin({
```

Comment:
Also please wrap this in deviceAccess request call https://github.com/trezor/trezor-suite/blob/af34896eb90028cc6b7a43795772dcc61e107bd8/suite-native/device-mutex/src/requestDeviceAccess.ts. All the device interaction of suite-native should use it to avoid concurrent access to the device memory.

Reply:
Fixed, thanks.